config/locales/en.yml

```diff
@@ -50,6 +50,7 @@ en:
     apply_from_find_with_ucas: Apply for this course with UCAS
     apply_to_course_on_ucas: You need to apply to this course on UCAS
     apply_to_provider_on_ucas: You need to apply to this provider on UCAS
+    unsubmitted_previous_application: Your unsubmitted application
```

Comment:
Should this be something like `Your previous application`? Now we have a separate view for the review_previous_application, but it is not necessarily unsubmitted

Reply:
Possibly one for @Tomcowan1. For a previously *submitted* application we say ‘Your submitted application’, correct? (Can offer context if needed Tom)

Reply:
We can potentially go back to sharing the view and render submitted vs unsubmitted application depending on what its state is. Although "Your unsubmitted application" feels a bit ambiguous as it could refer to the current one.

Reply:
So 'Your previous unsubmitted aplication' would work? Or is it too verbose

Reply:
Hi @richardpattinson Richard - can I just check the scenario - is an unsubmitted application only for people who carry over from one cycle to another?

Reply:
If the candidate has made a previous application and didnt submit it then they should be able to view this page. In practice that should only happen when they carry over from a previous year.

Reply:
Thanks @richardpattinson. I'm happy to go with "Your previously unsubmitted application." We need to differentiate it from the current one (that's also unsubmitted).